src/components/MenuEnvelope.vue

```diff
@@ -1,187 +1,183 @@
-			event=""
-			@click.native.prevent>
-			<template v-if="!moreActionsOpen">
-				<ActionButton v-if="hasWriteAcl"
-					class="action--primary"
-					:close-after-click="true"
-					@click.prevent="onToggleImportant">
-					<template #icon>
-						<ImportantIcon
-							:size="20" />
-					</template>
-					{{
-						isImportant ? t('mail', 'Unimportant') : t('mail', 'Important')
-					}}
-				</ActionButton>
-				<ActionButton v-if="withReply"
-					:close-after-click="true"
-					@click="onReply">
-					<template #icon>
-						<ReplyAllIcon v-if="hasMultipleRecipients"
-							:title="t('mail', 'Reply all')"
-							:size="20" />
-						<ReplyIcon v-else
-							:title="t('mail', 'Reply')"
-							:size="20" />
-					</template>
-					{{ t('mail', 'Reply') }}
-				</ActionButton>
-				<ActionButton v-if="hasMultipleRecipients"
-					:close-after-click="true"
-					@click="onReply(true)">
-					<template #icon>
-						<ReplyIcon
-							:title="t('mail', 'Reply to sender only')"
-							:size="20" />
-					</template>
-					{{ t('mail', 'Reply to sender only') }}
-				</ActionButton>
-				<ActionButton :close-after-click="true"
-					@click="onForward">
-					<template #icon>
-						<ShareIcon
-							:title="t('mail', 'Forward')"
-							:size="20" />
-					</template>
-					{{ t('mail', 'Forward') }}
-				</ActionButton>
-				<ActionButton v-if="hasWriteAcl"
-					:close-after-click="true"
-					@click.prevent="onToggleJunk">
-					<template #icon>
-						<AlertOctagonIcon
-							:title="envelope.flags.$junk ? t('mail', 'Mark not spam') : t('mail', 'Mark as spam')"
-							:size="20" />
-					</template>
-					{{
-						envelope.flags.$junk ? t('mail', 'Mark not spam') : t('mail', 'Mark as spam')
-					}}
-				</ActionButton>
-				<ActionButton v-if="hasWriteAcl"
-					:close-after-click="true"
-					@click.prevent="onOpenTagModal">
-					<template #icon>
-						<TagIcon
-							:title="t('mail', 'Edit tags')"
-							:size="20" />
-					</template>
-					{{ t('mail', 'Edit tags') }}
-				</ActionButton>
-				<ActionButton v-if="withSelect"
-					:close-after-click="true"
-					@click.prevent="toggleSelected">
-					<template #icon>
-						<CheckIcon
-							:title="isSelected ? t('mail', 'Unselect') : t('mail', 'Select')"
-							:size="20" />
-					</template>
-					{{
-						isSelected ? t('mail', 'Unselect') : t('mail', 'Select')
-					}}
-				</ActionButton>
-				<ActionButton
-					v-if="hasDeleteAcl"
-					:close-after-click="true"
-					@click.prevent="onOpenMoveModal">
-					<template #icon>
-						<OpenInNewIcon
-							:title="t('mail', 'Move message')"
-							:size="20" />
-					</template>
-					{{ t('mail', 'Move message') }}
-				</ActionButton>
-				<ActionButton :close-after-click="false"
-					@click="moreActionsOpen=true">
-					<template #icon>
-						<DotsHorizontalIcon
-							:title="t('mail', 'More actions')"
-							:size="20" />
-					</template>
+	<Fragment>
+		<template v-if="!moreActionsOpen">
```

Comment:
You still need the actions component wrapping the action items!

Reply:
this will just duplicate the primary actions outside the action menu, i have the actions on threadenvelope. And it was working fine, till i did the rebase and pulled on server.